.shippable.yml

```diff
@@ -32,6 +32,13 @@ build:
         options: "-e HOME=/home/buildslave --privileged=true --tty --net=bridge --user buildslave"
 
     ci:
+      - cd ..
+      - west init .
```

Comment:
So here you will be fetching whatever the default manifest says?
With this, we do not know what is actually run in CI anymore (it depends on a external repo that changes on its own)
== Not ok.*1

To do this properly, you would need to fetch from here a particular version of a particular manifest.
It should be a commit on its own to change which manifest version is used.

*1 Just think about the case when you will be moving some ext/ component to a external repo. You will on one side change the manifest, while on Zephyr you will do a PR to remove that ext/ component and change how its built. While you have changed the manifest and until the relevant changes have been merged in Zephyr, Zephyr will be broken.

Reply:
Yes today. But once you start using for what it is meant to be used : handling multiple repositories it needs to be handled as I commented.